database/src/main/postgres/flows/V1.9.1__get_flow_checkpoints_v2.sql

```diff
@@ -66,53 +70,60 @@ $$
 --      measured_columns       - measure columns associated with a given checkpoint
 --      measurement_value      - measurement details associated with a given checkpoint
 --      checkpoint_time        - time
+--      has_more               - flag indicating whether there are more checkpoints available
 --
 -- Status codes:
---      11                  - OK
---      41                  - Partitioning not found
---
--------------------------------------------------------------------------------
-
+--      11                     - OK
+--      42                     - Flow not found
+---------------------------------------------------------------------------------------------------
 DECLARE
-    _fk_partitioning BIGINT;
-    _fk_flow BIGINT;
+    _actual_limit INT := i_limit + 1;
```

Comment:
Why don't you follow the same approach as in other PRs? It's been discussed and we agreed collectively to do it that way.

Reply:
Will do that, thanks. Was trying to explore if it could work better, and reduce complexity in our logic.